docs/reference/snapshot-restore/apis/restore-snapshot-api.asciidoc

```diff
@@ -134,10 +134,22 @@ The cluster state includes:
 * Legacy index templates
 * Ingest pipelines
 * {ilm-init} lifecycle policies
+* For snapshots taken after 7.12.0, data stored in system indices, such as Watches and task records, replacing any existing configuration (configurable via `feature_states`)
```

Comment:
same comment regarding the docs indicating that this exists in the cluster state.